open-apparel-registry/open-apparel-registry
# Consolidate Contributor Type options

## Overview

In an effort to streamline and organize across our systems,
consolidates our current Contributor Types and updates existing
contributor's contrib_types:

From:
Auditor
Brand / Retailer
Civil Society Organization
Factory / Facility
Manufacturing Group / Vendor / Supplier
Journalist
Multi Stakeholder Initiative
Researcher / Academic
Service Provider
Student
Supplier
Union
Other

To:
Academic / Researcher / Journalist / Student
Auditor / Certification Scheme / Service Provider
Brand / Retailer
Civil Society Organisation
Facility / Factory / Manufacturing Group / Supplier / Vendor
Multi-Stakeholder Initiative
Union
Other

Connects #1359 

## Demo

Before:
<img width="613" alt="Screen Shot 2021-06-10 at 4 00 28 PM" src="https://user-images.githubusercontent.com/21046714/121725515-5c390a00-cab7-11eb-99f9-fcfa04a11d8c.png">
After:
<img width="611" alt="Screen Shot 2021-06-10 at 4 00 50 PM" src="https://user-images.githubusercontent.com/21046714/121725521-5e02cd80-cab7-11eb-9f67-124a43a13d86.png">

## Testing Instructions

* Run `./scripts/manage migrate`
* Run `./scripts/manage dbshell` and `SELECT contrib_type FROM api_contributor GROUP BY contrib_type;` and you will see only new contributor types
* Run `./scripts/server` then [register](http://localhost:6543/auth/register) and confirm that the contributor types select contains the new types

## Checklist
